presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -271,6 +276,87 @@ After launching, you can find the log files in ``var/log``:
   This is the HTTP request log which contains every HTTP request
   received by the server. It is automatically rotated and compressed.
 
+An Example Deployment on Laptop
```

Comment:
could it be: `An Example deployment to query data in S3`?

Reply:
done